src/module.c

```diff
@@ -13523,6 +13512,34 @@ void moduleDefragGlobals(void) {
     dictReleaseIterator(di);
 }
 
+/* Helper function for VM_Replicate and VM_ReplicateWithFlag to replicate the specified command
+ * and arguments to replicas and AOF, as effect of execution of the calling command implementation.
+ * Skip command validation if the ValkeyModuleFlag is set to VALKEYMODULE_FLAG_SKIP_VALIDATION. */
+int moduleReplicate(ValkeyModuleCtx *ctx, ValkeyModuleFlag flag, const char *cmdname, const char *fmt, va_list ap) {
```

Comment:
Pls move this function to just above the VM_Replicate, reference the function zsetInitLexRange. It is a helper function too.